config/routes.rb

```diff
@@ -4,6 +4,9 @@
         GovukHealthcheck::Mongoid,
       )
 
+  get "/healthcheck/live", to: proc { [200, {}, %w[OK]] }
+  get "/healthcheck/ready", to: GovukHealthcheck.rack_response
```

Comment:
GovukHealthcheck::Mongoid
```suggestion
  get "/healthcheck/ready", to: GovukHealthcheck.rack_response(
    GovukHealthcheck::Mongoid,
  )
```